src/Wizard.php

```diff
@@ -15,7 +15,7 @@ class Wizard extends View
     /**
      * Callback, that triggers selection of a step.
      *
-     * @var callable
+     * @var callback
```

Comment:
`callable` is old callable type which support strings, arrays, etc. But it should be actually `\Closure`

Reply:
`\Closure`! if that does not help, let me know, I will find a solution for sure

Reply:
You not `callable` nor `callback`, but `Callback`. Ok, I was reading this quite fast... I will find a solution, maybe it needs to be moved to the top of the phpdoc. I will fix shortly. I might also raise a ticket for this in CS fixer.